node-fs-readfile/cat.ts

```diff
@@ -0,0 +1,21 @@
+import * as fs from 'node:fs/promises';
+
+async function read(): Promise<void> {
+  try {
+    if (process.argv.length < 3) {
+      console.log('Invalid arguments.');
+      process.exit(1);
+    }
+
+    let result: string = '';
```

Comment:
It's good to put types in while you're learning to make sure you understand it all correctly. But as time goes on, you should rely more on TypeScript's ability to figure out types on its own. I usually leave off types and add them only when TS complains.